catalog-rest-service/src/main/java/org/openmetadata/catalog/events/ChangeEventHandler.java

```diff
@@ -186,8 +186,7 @@ private void handleWebSocket(ContainerResponseContext responseContext) {
                 });
             return;
           case Announcement:
-          default:
-            return;
+            WebSocketManager.getInstance().broadCastMessageToAll(WebSocketManager.announcementChannel, jsonThread);
```

Comment:
@parthp2107 An announcement is active only during its [scheduled time](https://github.com/open-metadata/OpenMetadata/blob/main/catalog-rest-service/src/main/resources/json/schema/entity/feed/thread.json#L123) (startTime - endTime)

Can we send notifications only if the announcement is active?

Reply:
Sure @vivekratnavel, will make the requested changes.

Reply:
@vivekratnavel, addressed in the latest commit.